pochivm/api_base.h

```diff
@@ -14,26 +14,8 @@ namespace PochiVM
 template<typename T>
 class Reference;
 
-// Const primitive reference: only used as an intermediate helper for calling C++ functions.
-// If a C++ function has parameter 'const int&', C++ allows both rvalue (e.g. '1') and reference (e.g. 'a')
-// to bind to that parameter. We need this helper struct to support this behavior.
-// If the input is rvalue, the conversion to this struct generates a 'AstRvalueToConstPrimitiveRefExpr' operator.
-//
 template<typename T>
-class ConstPrimitiveReference
```

Comment:
Can you explain what did you do in this commit? Is it undoing a previous commit you did? Since I cannot see its effect in the final "file changes"

Reply:
Earlier you highlighted the extra newline under the ConstPrimitiveReference forward declaration and told me to remove it. In the previous commit I misinterpreted it as you telling me to remove the whole ConstPrimitiveReference forward declaration and move the actual class up there. But then I realized you had only highlighted that one newline. So I restored the old forward declaration and just deleted that one newline.